lib/cog/command/gen_command/base.ex

```diff
@@ -173,8 +178,10 @@ defmodule Cog.Command.GenCommand.Base do
     attr_value(module, :command_name)
   end
 
-  def description(module),
-    do: attr_value(module, :description)
+  for attr <- [:description, :long_description, :examples, :notes, :arguments, :subcommands] do
```

Comment:
Guess you might as well pull `:command_name`, `:bundle_name`, `:rules`, and `:permissions` into this as well.